babel/messages/pofile.py

```diff
@@ -626,6 +626,11 @@ def _format_message(message, prefix=''):
 
             for filename, lineno in locations:
                 location = filename.replace(os.sep, '/')
+                if " " in location or "\t" in location:
+                    if not location.startswith("\u2068"):
+                        location = "\u2068" + location
+                    if not location.endswith("\u2069"):
+                        location += "\u2069"
```

Comment:
Could this be refactored into a separate module-internal helper function (`_maybe_enclose_filename` for instance?) that has references to the relevant Gettext code?

As it is, it's a bit non-obvious why this is here :)